designer/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentServiceImpl.scala

```diff
@@ -272,11 +266,13 @@ class DeploymentServiceImpl(dispatcher: DeploymentManagerDispatcher,
   }
 
   private def getStateFromEngine(deploymentManager: DeploymentManager, processIdWithName: ProcessIdWithName)
-                                (implicit ec: ExecutionContext): Future[Option[ProcessState]] =
-    deploymentManager.findJobStatus(processIdWithName.name).recover {
+                                (implicit ec: ExecutionContext, freshnessPolicy: DataFreshnessPolicy): Future[WithDataFreshnessStatus[Option[ProcessState]]] =
+    deploymentManager.getProcessState(processIdWithName.name).recover {
```

Comment:
we do not need `WithDataFreshnessStatus` here really?
can we do `deploymentManager.getProcessState(processIdWithName.name).map(_.value)` ?